pkg/relay/processmanager.go

```diff
@@ -0,0 +1,173 @@
+package relay
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/blocknative/dreamboat/pkg/structs"
+	"github.com/flashbots/go-boost-utils/bls"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type ProcessManager struct {
+	M   map[string]uint64
```

Comment:
`M`, `acc` should have readable names, in order to understand their purpose. 

Also, what's the difference between Stack `A`, `B` and `C`? I think the name should make this clear. It's important variable naming in public repositories

Reply:
If `A`, `B` and `C` are random channels, then we should put the explanation here (on top) instead of inside `VerifyParallel`

Reply:
However, I think names should be something similar to `VerifySubmitBlockCh`, `VerifyRegisterValidatorCh` and `VerifyOtherCh`

Reply:
https://github.com/blocknative/dreamboat/pull/37/files/b46041f077c62cfb41dc90b9f26304def179d781#diff-0a50202f9351d5a88dd5428ef1ce9c8a264beff7ce7e7662f50b567dd630d073R142 

The names are A,B,C - as it's not processed in priority order, as you've said it's hard to say which one is really more important to process.  
The idea for that, was also so that package should be mostly unaware of what's inside and don't care about the payloads as every one is processed in the same way. I might have used something like containers.Ring, but this would have been more complicated. So kept this just as hardcoded select with those 3.

Reply:
I changed the name to your suggestions also M and acc to something more desciptive.